deploy/taxbrain_server/celery_tasks.py

```diff
@@ -70,7 +70,6 @@ def dropq_task(year, user_mods, first_budget_year, beh_params, tax_data):
                     user_mods[reform_year].pop(key)
         user_reform = {"policy": user_mods}
     print('user_reform', user_reform, user_mods)
-    reform_style = [True if x else False for x in user_reform]
     if beh_params:
```

Comment:
`reform_style` can only be `[True]` since it is a dictionary with just one key `policy`.  Further, we now pass back assumptions data with empty dictionaries even if no assumptions are set.  So, this method no longer detected whether the user specified assumptions or not.